test/parallel/test_torch.py

```diff
@@ -2088,8 +2087,6 @@ def check_file(fname, check_cycle=True):
                 with open(fname, 'r') as timeline_file:
                     timeline_text = timeline_file.read()
                     assert 'allreduce.test_allreduce' in timeline_text, timeline_text
-                    assert 'NEGOTIATE_ALLREDUCE' in timeline_text, timeline_text
```

Comment:
Why are we not checking for these anymore?

Reply:
Negotiate all reduce is not there in these tests - 

https://travis-ci.org/github/horovod/horovod/jobs/732304151

Reply:
Hmm, that seems wrong. We should always expect to see NEGOTIATE followed by the actual ALLREDUCE.

Reply:
Not sure if there is existing problem? I still don't see NEGOTIATE  ALLREDUCE, can you check if this test  mpi set up, is logging NEGOTIATE ALL REDUCE if timeline is enabled from start.
https://travis-ci.org/github/horovod/horovod/jobs/733822943